lib/tpm_kdfa.c

```diff
@@ -51,6 +51,34 @@ static const EVP_MD *tpm_algorithm_to_openssl_digest(TPMI_ALG_HASH algorithm) {
     /* no return, not possible */
 }
 
+static HMAC_CTX *hmac_alloc()
+{
```

Comment:
FYI not aligned with coding standard here, see CONTRIBUTING. But I'll take it anyways.